src/views/Events/Events.module.scss

```diff
@@ -0,0 +1,26 @@
+_searchPageTitle {
+  color: var(--mui-palette-secondary-main);
+}
+_event_filter_icon {
+  color: var(--mui-palette-neutral-900);
+  font-size: 20;
+}
+_event_filter_list_icon {
+  color: var(--mui-palette-neutral-900);
```

Comment:
Did you test this color?  I don't think our neutral 900 is the same as the old gray 900, so I just wanted to make sure you looked at it